pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_per_pr_jobs.go

```diff
@@ -9,12 +9,19 @@ import (
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
-func GetAggregationIDFromProwJob(prowJob *prowjobv1.ProwJob) string {
-	return prowJob.Labels["release.openshift.io/aggregation-id"]
+const (
+	// AggregationIDLabel is the name of the label for the aggregation id in prow job
+	AggregationIDLabel = "release.openshift.io/aggregation-id"
+	// PayloadInvocationIDLabel is the name of the label for the payload invocation id in prow job
+	PayloadInvocationIDLabel = "release.openshift.io/aggregation-id"
```

Comment:
For now the payload invocation label is set to "release.openshift.io/aggregation-id" just for testing purpose. When this is set by the /payload implementation, we can change it to the matching value.